components/reference-data/src/main/java/org/datacleaner/beans/transform/RemoveDictionaryMatchesTransformer.java

```diff
@@ -79,14 +108,21 @@ public RemoveDictionaryMatchesTransformer(InputColumn<String> column, Dictionary
     @Override
     public OutputColumns getOutputColumns() {
         final String name = _column.getName() + " (" + _dictionary.getName() + " removed)";
-        return new OutputColumns(String.class, new String[] { name });
+        switch (_removedMatches) {
+        case LIST:
+            return new OutputColumns(new String[] { name, OUTPUT_COLUMN_REMOVED_MATCHES }, new Class[] { String.class,
+                    List.class });
+        default:
```

Comment:
While the meaning is the same (right now), using default instead of a concrete value isn't very easy on the reader of the code. Consider (in general, not just here) that you have two settings, but tomorrow you might have three. So arrange the code as if they are adjacent options instead of an "if else" kind of structure.